nss-day-cohort-39/tabloid-full-stack-coulees
# search posts

### What I did:
- Set up SQL in PostRepository to search multiple words in any order for tags, category, content, and UserDisplayName
- Set up a search bar in header to search the posts

### How to test:
- Run searches with multiple words on Post to see that it filters down from Post view
- Try searching using upper and lower cases to ensure this works 

*Note: Special character problem when searching #, but approved to pass off if this works
